addons/web/static/src/core/model_field_selector/model_field_selector.js

```diff
@@ -91,6 +91,7 @@ export class ModelFieldSelector extends Component {
             },
             {
                 closeOnClickAway: true,
+                popoverClass: "o-popover-no-arrow",
```

Comment:
This seems weird that the o-popover--no-arrow class is defined in the popoverscss file without being used directly by the popover component itself

why not pass another option instead of popoverClass: "withoutPointingArrow"  or smtg like that ?
The popover could then add a t-if attribute on its arrow el

Reply:
Definitely better, here for the moment, it seems like a "hack".
Will improve this